couchexport/groupexports.py

```diff
@@ -18,12 +18,22 @@ def export_for_group(export_id_or_group, output_dir):
 
     for subconfig, schema in config.all_exports:
         try:
-            rebuild_export(subconfig, schema, output_dir)
+            rebuild_export(subconfig, schema, output_dir, ignore_unused=ignore_unused)
         except ExportRebuildError:
             continue
 
 
-def rebuild_export(config, schema, output_dir):
+def rebuild_export(config, schema, output_dir, ignore_unused=False):
+    if output_dir == "couch":
+        saved = SavedBasicExport.view("couchexport/saved_exports",
+                                      key=json.dumps(config.index),
+                                      include_docs=True,
+                                      reduce=False).one()
+        if ignore_unused and saved and saved.last_accessed and \
+                (datetime.utcnow() - saved.last_accessed) > timedelta(days=7):
```

Comment:
rather than hardcoding this it could make sense to make `ignore_unused` a timestamp or a timedelta (and maybe rename) and treat `None` like `False` to preserve the default behavior.